### ISSUE TITLE
Densenet (Floating point models) reported accuracy is much lower than the original paper

### ISSUE THREAD

**Issue by cmhungsteve** [tensorflow/tensorflow#44032]
## URL(s) with the issue:
https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models

## Description of issue (what needs changing):
The accuracy of Densenet reported in the URL is much lower than the one reported in the original paper.

### Clear description
In the original paper (https://arxiv.org/abs/1608.06993), the performance is as follows:
Top-1: 74.98%; Top-5: 92.29% (Densenet-121)
However, the performance reported in the URL is as follows:
Top-1: 64.2%; Top-5: 85.6% (Densenet-?)
And there is no information about the depth of the provided densenet model.

Is there any modification for this model?
Or they are typos?

**Comment by Saduf2019** [comment]
@cmhungsteve 
Please share a simple stand alone indented code to replicate the issue faced or a colab gist with the issue.

**Comment by cmhungsteve** [comment]
It's not about the code. I guessed it's about the typos on the website.

I found that the accuracy numbers of Densenet on the website (https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) are very different from the Densenet paper (https://arxiv.org/abs/1608.06993).
I wonder the reason for the inconsistency.
Thank you.

**Comment by jdduke** [comment]
Apologies for the delayed response, this completely missed my radar.

I actually don't know the origin of the TFLite-reported DenseNet numbers, @miaout17 are you familiar with those numbers? Are we still tracking it? It seems to be fairly dated, we might consider removing it completely.

**Comment by mohantym** [comment]
Hi @cmhungsteve !
Concerned Documentation in the template is updated now. You can find the respective pre-trained models from TF-hub and TF-examples now.

Please refer performance [benchmark section](https://www.tensorflow.org/lite/examples/object_detection/overview#performance_benchmarks) in the examples for performance and accurancy related details.

Thank you!
